open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -65,8 +65,8 @@
         </dependency>
 
         <dependency>
-            <groupId>com.h2database</groupId>
-            <artifactId>h2</artifactId>
+            <groupId>org.hsqldb</groupId>
```

Comment:
Why hsqldb?

Reply:
H2 new version has compatibility issues

Reply:
and it's a toy database anyway used only for demo purpose

Reply:
is hsqldb not acceptable ?